staging/src/k8s.io/kube-scheduler/config/v1alpha2/types_pluginargs.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+// InterPodAffinityArgs holds arguments used to configure the InterPodAffinity plugin.
+type InterPodAffinityArgs struct {
+	// HardPodAffinityWeight is the scoring weight for existing pods with a
+	// matching hard affinity to the incoming pod.
+	HardPodAffinityWeight *int32 `json:"hardPodAffinityWeight,omitempty"`
+}
+
+// NodeLabelArgs holds arguments that used to configure the NodeLabel plugin.
+type NodeLabelArgs struct {
+	// PresentLabels should be present for the node to be considered a fit for hosting the pod
+	// +listType=atomic
+	PresentLabels []string `json:"presentLabels,omitempty"`
+	// AbsentLabels should be absent for the node to be considered a fit for hosting the pod
+	// +listType=atomic
+	AbsentLabels []string `json:"absentLabels,omitempty"`
+	// Nodes that have labels in the list will get a higher score.
+	// +listType=atomic
+	PresentLabelsPreference []string `json:"presentLabelsPreference,omitempty"`
+	// Nodes that don't have labels in the list will get a higher score.
+	// +listType=atomic
+	AbsentLabelsPreference []string `json:"absentLabelsPreference,omitempty"`
+}
+
+// NodeResourcesFitArgs holds arguments used to configure the NodeResourcesFit plugin.
+type NodeResourcesFitArgs struct {
+	// IgnoredResources is the list of resources that NodeResources fit filter
+	// should ignore.
+	// +listType=atomic
+	IgnoredResources []string `json:"ignoredResources,omitempty"`
+}
+
+// RequestedToCapacityRatioArgs holds arguments used to configure RequestedToCapacityRatio plugin.
+type RequestedToCapacityRatioArgs struct {
+	// Points defining priority function shape
+	// +listType=atomic
+	Shape []UtilizationShapePoint `json:"shape"`
```

Comment:
Maybe it's ok to use the v1 type here. Again, for api reviewers.

Reply:
Good call. It doesn't seem it can diverge.